community-members.md

```diff
@@ -64,11 +64,7 @@ Find out active members - approvers and maintainers of .NET SDK in [open-telemet
 
 Repo: [open-telemetry/opentelemetry-go](https://github.com/open-telemetry/opentelemetry-go)
 
-Approvers:
-
-- [Paulo Janotti](https://github.com/pjanotti), Splunk
-- [Steven Karis](https://github.com/sjkaris), Splunk
-- [Ted Young](https://github.com/tedsuo), LightStep
+Approvers: see the current [CODEOWNERS](https://github.com/open-telemetry/opentelemetry-go/blob/master/CODEOWNERS) file.
```

Comment:
There are three lists for approvers:
- Approvers with the company affiliation. Ideally place it in readme.
- Members of the team like @open-telemetry/go-approvers 
- CODEOWNERS

Note, that approvers and CODEOWNERS file will not match. First, maintainers will also be there. Second, triagers will be approvers, and part of a team, but not in CODEOWNERS file.

So I'd suggest to add this list to readme and point from this file to the readme of the appropriate repo. As done for couple other SIGs already in this file.

Reply:
done.